tests/queries/0_stateless/02161_addressToLineWithInlines.sql

```diff
@@ -0,0 +1,25 @@
+-- Tags: no-tsan, no-asan, no-ubsan, no-msan, no-debug
+
+
+SELECT addressToLineWithInlines(1); -- { serverError 446 }
+
+SET allow_introspection_functions = 1;
+SET query_profiler_real_time_period_ns = 0;
+SET query_profiler_cpu_time_period_ns = 1000000;
+SET log_queries = 1;
+SELECT count() FROM numbers_mt(10000000000) SETTINGS log_comment='02161_test_case';
+SET log_queries = 0;
+SET query_profiler_cpu_time_period_ns = 0;
+SYSTEM FLUSH LOGS;
+
+WITH
+    lineWithInlines AS
+    (
+        SELECT DISTINCT addressToLineWithInlines(arrayJoin(trace)) AS lineWithInlines FROM system.trace_log WHERE query_id =
+        (
+            SELECT query_id FROM system.query_log WHERE current_database = currentDatabase() AND log_comment='02161_test_case' ORDER BY event_time DESC LIMIT 1
+        )
+    )
+SELECT 'has inlines:', or(max(length(lineWithInlines)) > 1, not any(locate(lineWithInlines[1], ':') != 0)) FROM lineWithInlines SETTINGS short_circuit_function_evaluation='enable';
```

Comment:
I guess, there should be `max(locate(lineWithInlines[1], ':')) = 0` instead of `not any(locate(lineWithInlines[1], ':') != 0)`, if you mean "in all rows `locate(lineWithInlines[1], ':')` returns 0", because function `any` just returns one value from random row.

Reply:
Got it, I misunderstood the behavior of `any`, I will fix.